lib/main.dart

```diff
@@ -25,6 +25,7 @@ class SurveyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle.light);
```

Comment:
We already set the `systemUIOverlayStyle` inside `_setTransparentStatusBar()`.

What about updating this? 🙏 

```

void _setTransparentStatusBar() {
  SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle.light.copyWith(
    statusBarColor: Colors.transparent,
  ));
}
```